cli/src/remote/cozy.js

```diff
@@ -74,15 +74,16 @@ export default class RemoteCozy {
 
   trashById: (id: string, options?: {ifMatch: string}) => Promise<RemoteDoc>
 
-  async changes (seq: number = 0) {
-    let json = await this.client.data.changesFeed(FILES_DOCTYPE, { since: seq })
+  async changes (since: string = '0'): Promise<{last_seq: string, docs: Array<RemoteDoc|RemoteDeletion>}> {
+    const options = {since, include_docs: true}
+    const {last_seq, results} = await this.client.data.changesFeed(FILES_DOCTYPE, options)
+    let docs = results.filter(r => !specialId(r.id)).map(r => r.doc)
 
-    return {
-      last_seq: json.last_seq,
-      ids: json.results
-        .map(result => result.id)
-        .filter(id => !specialId(id))
+    for (const doc of docs) {
+      if (doc.type === 'file') await this._setPath(doc)
```

Comment:
@nono Maybe we could have retrieved all parent dirs at once...
Not sure which index we should use though... Just `_id`? Passing the parent ids (without duplicates) as a mango query? WDYT?